File: nodejs/nestjs/src/services/prime/prime.service.ts
import { Injectable } from '@nestjs/common';

const { v4 } = require('uuid');
const {
  prime: { eratosthenes },
} = require('utils');

@Injectable()
export class PrimeService {
  public process(top: number): string {
    return eratosthenes(top);
  }
}

File: stacks/java/spring-boot-tomcat/src/main/java/co/com/bancolombia/service/PrimeService.java
package co.com.bancolombia.service;

import co.com.bancolombia.utils.Prime;
import org.springframework.stereotype.Service;

@Service
public class PrimeService {
    private final Prime prime = new Prime();

    public String process(int top) {
        return prime.primo(top);
    }
}
